packages/odyssey-react-mui/package.json

```diff
@@ -58,8 +59,6 @@
   "peerDependencies": {
     "@emotion/react": "^11",
     "@emotion/styled": "^11",
-    "@mui/icons-material": "^5",
```

Comment:
why the move from peerDependencies to dependencies?

Reply:
That's because we're in charge of installing these now. Think of Odyssey as MUI agnostic. It just _happens_ to use it today.

I plan to add an [`optionalDependency`](https://docs.npmjs.com/cli/v8/configuring-npm/package-json#optionaldependencies) that'll show up if a consumer pulls in their own MUI.

Reply:
@KevinGhadyani-Okta Does tree shaking still work for consumers of odyssey-react-mui all the way to MUI 5 dependencies? This was a large pain point with MUI 4 (bad/no-tree shaking) which was mostly addressed in MUI 5. But I'm not confident it will also work with an indirect dependency like this (iga -> odyssey-react-mui -> mui-5).

Reply:
As far as I understand it, with `sideEffects: false`, which we support, it should be able to properly tree-shake MUI components with the way we're using ESM exports.

The only reason you wouldn't get proper exports is if MUI is setup as CJS. As ESM, tree-shaking should work properly, and MUI v5 has ESM from my look at their source.

Reply:
`optionalDependencies` doesn't work the way I expected. I reverted that change. At this point, I may have to add that info to the README.

Reply:
Can you try a quick test locally? If for some reason going to a hard dependency here does not allow dependents to tree shake, I would rather you had a "mandatory peer pegged at a specific mui version" that allowed me to tree shake then offer the convenience of me not having to install mui but no tree shaking.

Reply:
I could quickly test in Example-React-SPA if we have one for Vite.

If we have a project using Webpack, the [webpack-bundle-analyzer](https://www.npmjs.com/package/webpack-bundle-analyzer) helps you see if tree-shaking works.

If MUI doesn't tree-shake from our library, it's not gonna change if you install MUI yourself because it's being imported from `odyssey-react-mui` now.